vividus-extension-selenium/src/main/java/org/vividus/selenium/screenshot/ScreenshotTaker.java

```diff
@@ -25,7 +24,5 @@
 {
     Optional<Screenshot> takeScreenshot(String screenshotName);
 
-    BufferedImage takeViewportScreenshot() throws IOException;
```

Comment:
Why was it removed?

Reply:
- screenshot taker works around ashot strategies, if we want to take viewport we use simple strategy
- we used this method in places that are not coupled to ashot so consider this as kind of low level api

Reply:
Why not have a single class responsible for taking the screenshots according to the global settings?

Reply:
Why not have a single class responsible for taking the screenshots according to the global settings?

Reply:
discussed privately